Branch-SDK/src/io/branch/referral/BranchRemoteInterface.java

```diff
@@ -86,7 +86,9 @@ public void connectToDebug() {
             post.put(Defines.Jsonkey.OSVersion.getKey(), sysObserver_.getOSVersion());
             post.put(Defines.Jsonkey.Model.getKey(), sysObserver_.getPhoneModel());
             post.put("is_simulator", sysObserver_.isSimulator());
-            callback_.finished(make_restful_post(post, prefHelper_.getAPIBaseUrl() + urlExtend, PrefHelper.REQ_TAG_DEBUG_CONNECT, prefHelper_.getTimeout(), false));
+            post.put(Defines.Jsonkey.SessionID.getKey(), prefHelper_.getSessionID());
```

Comment:
Don't we need to worry about whether this has been set or not? What happens in poor connectivity cases?